config_quick_debug.yml

```diff
@@ -10,14 +10,12 @@ scenario_params_list:
     - [0.2, 0.5, 0.3]
    samples_split_option:
      - ['basic', 'random']
-     - ['basic', 'stratified']
-     - ['advanced', [[4, 'shared'], [6, 'shared'], [4, 'specific']]]
    multi_partner_learning_approach:
     - 'fedavg'
    aggregation_weighting:
     - 'uniform'
    methods:
-    - ["Shapley values", "Independent scores"]
+    - ["TMCS", "PVRL"]
```

Comment:
Is this file supposed to evolve?

Reply:
We said thuesday that  the config_debug can change now, as we will create a config_travis. I may have anticipated this change here. I will put config_debug back as it was.